cmd/az-log/config/az-log.yaml

```diff
@@ -0,0 +1,4 @@
+kubeconfig: "" # default is "$HOME/.kube/config"
+driverNamespace: "" # default is "azure-disk-csi"
```

Comment:
Are we using the driverNamespace anywhere in the code?

Reply:
Yes, it is used for searching for lease of controller plugin.